docs: Skip password promot during cosign key generation (#131)
ublue-os/image-template: README.md

With the cosign tool installed, run inside your repo folder:

```bash
COSIGN_PASSWORD="" cosign generate-key-pair
```

> Do NOT put in a password when it asks you to, just press enter. The signing key will be used in GitHub Actions and will not work if it is encrypted.

Reviewer: ```suggestion
The signing key will be used in GitHub Actions and will not work if it is encrypted with a password.
```
might as well remove this part I guess
